contracts/mocks/DCAStrategies/DCAStrategiesPositionsHandlerMock.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.7 <0.9.0;
+
+import '../../DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol';
+
+contract DCAStrategiesPositionsHandlerMock is DCAStrategiesPositionsHandler {
+  struct CreateCall {
+    address owner;
+    IDCAStrategies.PermissionSet[] permissionSets;
+  }
+
+  CreateCall[] private _createCalls;
+  IDCAStrategies.ShareOfToken[] private _tokenShares;
+
+  function getCreateCalls() external view returns (CreateCall[] memory) {
+    return _createCalls;
+  }
+
+  function setTokenShares(IDCAStrategies.ShareOfToken[] calldata _tokens) external {
+    _tokenShares.push();
+    for (uint256 i = 0; i < _tokens.length; i++) {
+      _tokenShares[_tokenShares.length - 1].token = _tokens[i].token;
+      _tokenShares[_tokenShares.length - 1].share = _tokens[i].share;
+    }
```

Comment:
When you have an array of a struct that inside has an array, you need to do this. You need to push something to the first position, and then fill the array inside the struct (like in `_create`). But here, you just have a normal array
```suggestion
    for (uint256 i = 0; i < _tokens.length; i++) {
      _tokenShares.push(_tokens[i]);
    }
```